plugins/rplanners/ParabolicPathSmooth/ppramp.cpp

```diff
@@ -352,7 +352,7 @@ Real PPRamp::CalcTotalTime(Real a) const
 
 int PPRamp::CalcTotalTimes(Real a,Real& t1,Real& t2) const
 {
-    Real ts1,ts2;
+    Real ts1 = 0.0, ts2 = 0.0;
```

Comment:
Here too.

Reply:
what do you think about line 432 -- this was made before me:
```
Real PPRamp::CalcSwitchTime(Real a) const
{
    Real t1 = 0.0, t2 = 0.0;
    int res = CalcSwitchTimes(a,t1,t2);
```

Reply:
@Puttichai We need to clarify: CalcSwitchTime one has committed by LiBoon in 7a528ba721. Should that be fixed to -1.0?

Reply:
I checked `res = CalcSwitchTimes(a, t1, t2)`. The only case when `t1` and `t2` are not set to any values inside the function is when the return value `res` is `0`, meaning that there are no solutions.
I still think it makes sense to change the default values to `-1`.

Reply:
changed to -1